frontend/include/chpl/types/DomainType.h

```diff
@@ -99,8 +99,9 @@ class DomainType final : public CompositeType {
 
   /** Return an associative domain type */
   static const DomainType* getAssociativeType(Context* context,
-                                              const QualifiedType& idxType,
-                                              const QualifiedType& parSafe);
+                                 const QualifiedType& instance,
+                                 const QualifiedType& idxType,
+                                 const QualifiedType& parSafe);
```

Comment:
This looks misaligned according to our style. I believe the "proper" style would be to re-align decls of `instance` with the decl for `context`.

Less "proper", but what I've been doing, is to put `getAssociativeType` on a new line, and still re-align the formals. This saves a little bit of horizontal space.